sidebars.js

```diff
@@ -62,34 +62,28 @@ module.exports = {
               label: 'Reference 🔗',
               href: 'https://pkg.go.dev/go.temporal.io/sdk',
             },
-            {
```

Comment:
So in general I like this version far more than the triple nested thing. That being said, having everything in a list makes it sort of hard to differentiate.